### PR TITLE
Account onboarding/billing activation/ma 1397

### PR DESCRIPTION
**Pull request by chandramishra**
added one more flow in OnboardAccount workflow for Activate billing

### CODE REVIEW

**Review by bhargavapratik — APPROVED** [review]

**Comment by RajkumarShivage** [comment]
Change the target branch to develop

**Review by RajkumarShivage — APPROVED** [review]
LGTM

File: src/index.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ baseHandler.post = function(params, callback) {
   inputDoc.billing_master.roles = params.roles_to_federate_to_billing_master;
   inputDoc.configrules.rules = params.default_configrules_to_enable;
   inputDoc.configrules.customerAccount = params.account.id;
+  inputDoc.account.billingDetails = params.account;
```
Comment (SonalOjha) [inline src/index.js]: if this is account Id name it so

File: src/index_enable_billing_activation.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    masterAwsId: params.masterAwsId,
+    customerAwsId: accId,
+    customerAwsName: params.name,
+    type: params.type ,
+    awsDesc: params.desc,
+    activationDate: datetime
+  }
+  console.log(bodyjson);
+  var billingUrl = params.billingUrl+"?client_id="+ params.apiKey +"&client_secret="+params.secretKey;
+  console.log(billingUrl);
```
Comment (SonalOjha) [inline src/index_enable_billing_activation.js]: Its not recommended to log passwords or tokens which could be used.

**Review by SonalOjha — APPROVED** [review]
Approving

**Pull request merged**